### DISCUSSION TITLE
Changing theme to dark mode

### DISCUSSION THREAD

**Discussion by ZennerBlade** [ant-design/ant-design#39008] [Q&A]
Hi,

How do we change the theme to dark mode in Antd 5 (5.0.1)?

Earlier all we had to do was 
```JS
import 'antd/dist/antd.dark.css';
```

How are we doing it now?
If I use ConfigProvider and use the dark algorithm, it does make it dark to some extent but lot of texts still stay black which kind of makes it useless since you'd have to manually override them.

Prime example is the Empty component. The description of empty component stays in black text which is almost unreadable if you background is dark.

So, how are we switching to dark mode now?

**Comment by afc163** [comment]
We need a document for it.

**Reply by ZennerBlade** [reply]
How are we supposed to switch theme in the mean time? Manual styles?

For example, even with dark algorithm, the background for Notification still comes as white.
Now if you look at the value in theme editor, the attribute associated with notification's background color is 
```JS
colorBgElevated
```

But if you write the below code, it doesn't change anything, the background of notification still stays white.
Should this be a bug then?

```JS
theme={{
	algorithm: theme.darkAlgorithm,
	token: {
		colorTextBase: 'wheat',
		fontSizeHeading1: 48
	},
	components: {
		Notification: {
			colorBgElevated: 'red'
		}
	}
}}
````

**Reply by abenhamdine** [reply]
For Notification, you have to use `useNotification`  to see the theme applied, the docs will be updated, see https://github.com/ant-design/ant-design/pull/39068/files

**Reply by gabada** [reply]
Was this ever resolved? I am having the same issue. I am using v5 algorithms to change the theme but the dark theme doesn't change the background color.
